src/app/components/icons/IconSearchBar.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { ChangeEvent, useEffect, useState, HTMLAttributes, useCallback } from 'react';
+import { Search } from '@material-ui/icons';
+import { useQueryString } from '../../hooks/useQueryString';
+// Material-UI Components
+import {
+    Checkbox,
+    FormControl,
+    Grid,
+    InputAdornment,
+    InputLabel,
+    ListItemText,
+    makeStyles,
+    MenuItem,
+    // OutlinedInput,
+    Select,
+    TextField,
+    useMediaQuery,
+    // Theme,
+} from '@material-ui/core';
+
+// import * as Colors from '@pxblue/colors';
+import { titleCase } from '../../shared';
+
+const useStyles = makeStyles(() => ({
+    searchBar: {
+        display: 'flex',
+        flexWrap: 'wrap',
+    },
+    searchField: {
+        // flex: '1 1 0px',
+        // minWidth: 300,
+        // marginRight: theme.spacing(2)
+        // borderRadius: theme.shape.borderRadius,
+        // backgroundColor: Colors.white[50],
+        // color: theme.palette.text.primary,
+        // marginTop: theme.spacing(1),
+        // marginRight: theme.spacing(3),
+    },
+    categoryPicker: {
+        // flex: '1 1 0px',
+        // minWidth: 200,
+    },
+}));
+
+type SearchBarProps = HTMLAttributes<HTMLDivElement> & {
+    onSearchChange: (event: ChangeEvent<HTMLInputElement>) => void;
+    onCategoriesChanged: (event: ChangeEvent<{ name?: string; value: any }>) => void;
+    iconCategories: string[];
+};
+export const IconSearchBar: React.FC<SearchBarProps> = (props): JSX.Element => {
+    const { onSearchChange, onCategoriesChanged, iconCategories, ...divProps } = props;
+    const classes = useStyles();
+    const { iconSearch = '' } = useQueryString();
```

Comment:
Is this something within the scope of this PR? : 
When a search happens, should the URL be updated to include the search query string?
e.g. "http://localhost:3000/style/iconography?iconSearch=tree"

Reply:
I wondered the same thing. I checked the existing implementation and it doesn't do that, so I left it out for the time being.